src/components/RoomView/Card.tsx

```diff
@@ -18,7 +18,6 @@ const Card = (word: CardObj) => {
   const gameStatus = useSelector((state: RootState) => state.game.status);
   const assassinTeamId = useSelector((state: RootState) => state.assassinAndBystander.assassinTeamId);
   const bystanderTeamId = useSelector((state: RootState) => state.assassinAndBystander.bystanderTeamId);
-  const [teamsCard, setTeamsCard] = useState<number>(0);
```

Comment:
these pieces of state are likely an artifact from something before - they don't actually do anything. so i deleted them.